### ISSUE TITLE
remix 1.15.0 build (w/ express config) -- ~/routes/index renders blank page

### ISSUE THREAD

**Issue by rolanday** [remix-run/remix#5965]
### What version of Remix are you using?

1.15.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

1. `npx create-remix@latest`
2. choose express option
3. install deps
4. npm run dev
5. open `http://localhost:3000/` in browser
6. observe blank page (instead of usual sample content present in routes/index)

Here's package.json

```
{
  "private": true,
  "sideEffects": false,
  "scripts": {
    "build": "remix build",
    "dev": "npm-run-all build --parallel \"dev:*\"",
    "dev:node": "cross-env NODE_ENV=development nodemon --require dotenv/config ./server.js --watch ./server.js",
    "dev:remix": "remix watch",
    "start": "cross-env NODE_ENV=production node ./server.js",
    "typecheck": "tsc"
  },
  "dependencies": {
    "@remix-run/express": "^1.15.0",
    "@remix-run/node": "^1.15.0",
    "@remix-run/react": "^1.15.0",
    "compression": "^1.7.4",
    "cross-env": "^7.0.3",
    "express": "^4.18.2",
    "isbot": "^3.6.5",
    "morgan": "^1.10.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@remix-run/dev": "^1.15.0",
    "@remix-run/eslint-config": "^1.15.0",
    "@types/react": "^18.0.25",
    "@types/react-dom": "^18.0.8",
    "dotenv": "^16.0.3",
    "eslint": "^8.27.0",
    "nodemon": "^2.0.20",
    "npm-run-all": "^4.1.5",
    "typescript": "^4.8.4"
  },
  "engines": {
    "node": ">=14"
  }
}
```

And here's remix config
```
/** @type {import('@remix-run/dev').AppConfig} */
module.exports = {
  ignoredRouteFiles: ["**/.*"],
  // appDirectory: "app",
  // assetsBuildDirectory: "public/build",
  // serverBuildPath: "build/index.js",
  // publicPath: "/build/",
  future: {
    v2_errorBoundary: true,
    v2_meta: true,
    v2_normalizeFormMethod: true,
    v2_routeConvention: true,
  },
};
```

### Expected Behavior

To see content in routes/index rendered to page

### Actual Behavior

Blank page

**Comment by kiliman** [comment]
The template is wrong for `v2_routeConvention`

Rename `routes/index.tsx` to `routes/_index.tsx`

**Comment by MichaelDeBoey** [comment]
Fixed by #5969
